tests/pkcli/test2_test.py

```diff
@@ -22,4 +21,6 @@ def test_cases(capsys):
         o, e = capsys.readouterr()
         pkio.write_text("stdout.txt", o)
         pkio.write_text("stderr.txt", e)
-        pkunit.pkre(expect[d.basename], o)
+        pkunit.pkre(
+            'FAILED due to these "coroutine was never awaited" warnings:\n1\).+2\)', o
```

Comment:
Too much text. Again, failure here is a failing test. Make the regex looser so that if we change the text, the test doesn't necessarily have to be updated. One thing that definitely should be checked is that on line 20, that the test actually failed. You expect this. 

It's still unclear to me that the subprocess is necessary.

Reply:
Are you thinking test_subprocess is not necessary if we arent using the warnings summary anymore and just appending the relevant info to the end of the log?

Reply:
Yea, I think you might be able to get away with just running it inline.

Reply:
Ok. Should I remove test_subprocess since if it is run inline it doesn't really do anything different from test_coroutine_never_awaited?